src/components/gnb/GNB.jsx

```diff
@@ -6,18 +6,35 @@ import PageLink from './PageLink';
 import GnbDepth from './GnbDepth';
 
 const GNB = () => {
-  const [routerLink, setLink] = useState(['/best', '/agriculturalproducts', '/marineproducts', '/livestockproducts', '/processedfood', '/allproducts'])
-  const [page, setPage] = useState(['베스트', '농산물', '수산물', '축산물', '가공식품', '전체상품'])
+  // const [routerLink, setLink] = useState([
+  //   '/best',
+  //   '/agriculturalproducts',
+  //   '/marineproducts',
+  //   '/livestockproducts',
+  //   '/processedfood',
+  //   '/allproducts',
+  // ]);
+  // const [page, setPage] = useState(['베스트', '농산물', '수산물', '축산물', '가공식품', '전체상품']);
+  const LINKS = ['농산물', '수산물', '축산물', '가공식품', '식품명인', '전체상품'];
```

Comment:
```suggestion
  const links = ['농산물', '수산물', '축산물', '가공식품', '식품명인', '전체상품'];
```
로 해주세요..

제가 컨벤션 설명이 부족했던 것 같아요 죄송합니다 ㅜㅜ
상수명 전체에 대해서 대문자로 쓰는것은 특별한 상수임을 나타내는 것을 말합니다.
마치 해당 값의 '명사화' 라고 볼 수 있습니다.
이 변수명의 특징은 다음과 같습니다.
- 상수명이 전부 대문자로 쓰여서 **특별한 상수값**임을 강조함
- 해당 변수명과 내용이 거의 동치관계라 볼 수 있음
- 해당 자료(데이터)가 명사화 된 것임
- 여러 곳에서 호출되어 사용빈도가 높은 상수( 재활용성이 높아 사용빈도가 높음 )
- 절대 변하지 않는 값임을 명시함

차이를 이해하기 위해 예시를 들자면 이렇습니다.
```javascript
function printName( person ) { 
const { name } = person // 상수인 const name 이지만 person 값에 달라지는 값이므로 NAME 으로 쓰지 않음
console.log(name)
}
```

```javascript
const BASE_URL = 'https://asia-northeast3-heropy-api.cloudfunctions.net/api" // 프로젝트내에서 api호출 때마다 반복적으로 쓰이는 상수
const EMAIL = 'admin@gmail.com'   // 관리자의 폰번호

const signupURL = BASE_URL + "/auth/signup"  // 중요도가 높지 않고, 재활용하는 상수도 아님
let phoneNum = person.phone 
```
```javascript
const WINDOWS = 'windows' // 해당 변수와 값이 명사화. WINDOWS 와 'windows' 문자열은 동치관계라 볼 수 있음
const MAC = 'mac';

if ( computer.os === WINDOWS ) {
   console.log('윈도우입니다.')
}else if ( computer.os === MAC ){
  console.log('맥입니다')
}
```
정도의 예시를 들 수 있을거 같습니다.

따라서 해당 변수값은 `links` 만 해도 충분할 것 같습니다.

Reply:
넵 수정할게용

Reply:
네 이해했어요.. 저도 변수명 적으면서 이게 대문자로 적을만한건가 고민했지만,, 또 컨벤션에 따르면 대문자인거같고해서...ㅋㅋㅋ 무슨말씀인지 알겠습니다!!